src/protobuf/bmv2/table.proto

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Google LLC
```

Comment:
This is an overall comment, but I think it would be worth keeping all protos about representing bmv2 in a single file. It makes it much easier to read. What do you think?

Reply:
I am unsure if this increase readability, I am worried the file will be too big. I do not have a strong preference.